### PR TITLE
Wrapup

### PR DESCRIPTION
**Pull request by hcape**
notify and help sockets mostly

### CODE REVIEW

File: app/src/main/java/antidose/antidose/HelpActivity.java [hunk 1]
```diff
@@ -283,11 +285,18 @@ public void onMessage(String s) {
...
-                        //textView.setText(textView.getText() + "\n" + message);
-                        Button OTW = (Button) findViewById(R.id.buttonComing);
-                        updateOTWCount(OTW, message);
+                        if(message.equals("cancel")) {
+                            //toss it
+
+                        }else if(message.trim().isEmpty()) {
+                        //skip
+                        }else {
+                            //System.out.print(message);
```
Comment (JordanVlieg) [inline app/src/main/java/antidose/antidose/HelpActivity.java:294]: Commented code, can be removed?

**Review by graemebates — APPROVED** [review]
hope this works 👍

File: app/src/main/java/antidose/antidose/NotifyActivity.java [hunk 2]
```diff
@@ -216,38 +231,38 @@ public void onFailure(Call<RestInterface.IncidentLocation> call, Throwable t) {
...
-            }
-
-            @Override
-            public void onFailure(Call<RestInterface.NumberResponders> call, Throwable t) {
-                Log.d("D", "Getting number responders failed :(");
-                Log.d("D", t.toString());
-            }
-        });
-    }
+//    public void makeAPICallNumResponders(final TextView numComing, String token){
```
Comment (JordanVlieg) [inline app/src/main/java/antidose/antidose/NotifyActivity.java:234]: Commented code, can be deleted

**Review by JordanVlieg — APPROVED** [review]
Looks good, some minor cleanup could be done though.

**Pull request merged**